Firestore/core/src/local/local_serializer.cc

```diff
@@ -352,6 +350,8 @@ MutationBatch LocalSerializer::DecodeMutationBatch(
           transform_mutation.transform.field_transforms_count;
       new_mutation.update_transforms =
           transform_mutation.transform.field_transforms;
+      // Prevent double-freeing of the write's fields. The field are now owned
```

Comment:
Nit: `s/the field are/the fields are/` (applies throughout).

Reply:
Fixed